dts/bindings/flash_controller/st,stm32-ospi-nor.yaml

```diff
@@ -105,3 +105,7 @@ properties:
       * 2READ 1-2-2 (0xBB) -> 2READ 1-2-2 4B (0xBC)
       * QREAD 1-1-4 (0x6B) -> QREAD 1-1-4 4B (0x6C)
       * 4READ 1-4-4 (0xEB) -> 4READ 1-4-4 4B (0xEC)
+  block-erase-size:
+    type: int
+    description: |
+      Block erase size of the NOR-flash chip.
```

Comment:
As I understand the driver uses `0xD8` block erase command regardless of the attribute value.

In that case, won't it be better to have that as a Boolean flag to enable erasure in blocks?

Otherwise, some devices might support different block erase size which the driver won't be able to deliver (_i.e. 32KB Block Erase `0x52`_)